cpp/src/DataStorm/Instance.cpp

```diff
@@ -56,6 +56,7 @@ Instance::Instance(const Ice::CommunicatorPtr& communicator) : _communicator(com
     if (properties->getPropertyAsIntWithDefault("DataStorm.Node.Multicast.Enabled", 1) > 0)
     {
         properties->setProperty("DataStorm.Node.Adapters.Multicast.Endpoints", "udp -h 239.255.0.1 -p 10000");
+        properties->setProperty("DataStorm.Node.Adapters.Multicast.PublishedHost", "239.255.0.1");
```

Comment:
This is required because we use `ObjectAdapter::createProxy` to create the multicast proxy. Without this the multicast proxy would not contain the mcast address but the hostname from the default PublishedEndpoints.

Reply:
Might be worth having a comment on here.